Build-and-Deploy-Your-First-Dashboard.mdx

```diff
@@ -0,0 +1,420 @@
+
+
+
+Welcome to **Upsolve AI**, your go-to platform for enabling **embedded business intelligence** for your customers. By following this example we will walk you through the essentials of connecting your data, setting up tenancy, building your first dashboard, sharing your insights, and deploying your dashboards in your product.
+
+
+## Connecting data:
+
+
+We’ve already set up a **sample database** to help you quickly explore and test the platform without needing to connect your own data source. 
+
+To view this connection, please go to the **Connect** tab as seen below:
+
+
+<Frame>
+<img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/data-connection-hero.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/data-connection-hero-dark.png"
+  alt="Hero Dark"
+/>
+</Frame>
+
+If you would like to connect to a different data source, please have a look at how to [set up your connection](data-connection/data-connection).
+
+
+## Build your first dashboard
+
+In order to build your first dashboard you should begin by creating your first chart before adding to the dashboard. 
+
+
+### Build your first chart
+
+It requires five steps to build your first chart:
```

Comment:
Rewrite it to (will probably need to change some of the text below when we swap to a new database):

It requires five steps to build your first chart. In the below example, you will build a grouped bar chart using the **aggregations** table.